.travis.yml

```diff
@@ -3,26 +3,22 @@ sudo: false
 
 branches:
   only:
-    - master
+    #- master
+    - conda_install
 
 install:
   - source devtools/travis-ci/install.sh
-  - export PYTHONUNBUFFERED=true
-
-script:
   # Add omnia channel
   - conda config --add channels ${ORGNAME}
-  # Create and activate test environment
-  - conda create --yes -n test python=$python
-  - source activate test
-  # Build the recipe
+  - conda create -n testenv python=$python
+  - source activate testenv
   - conda build devtools/conda-recipe
-  # Install
-  - conda install --yes --use-local blues
-  # Run tests
-  - conda install --yes pytest
-  #- cd devtools && nosetests -vv --nocapture --with-timer $PACKAGENAME && cd ..
-  - cd blues/tests && pytest
+  - conda install blues --use-local
+  - conda list
```

Comment:
Hmm, `conda list` doesn't seem functional. Is this helpful for debugging output or something?